torch/csrc/jit/passes/quantization.cpp

```diff
@@ -962,15 +964,34 @@ graph(%self, %scale, %zero_point, %dtype):
     module.register_buffer(
         "_quantized_weight",
         at::quantize_per_tensor(float_weight, scale, zero_point, dtype));
+
+    // Replace the GetAttr[weight]->quantize_per_tensor sequence
```

Comment:
You can reuse the getattr node and just change the 'name' in it to point to "quantized_weight".